docs/README.md

```diff
@@ -0,0 +1,62 @@
+# ALBA
+ALBA(ALternative BAckend) is a distributed object store, like Swift, Ceph and Amplistor. ALBA is highly optimised for Open vStorage as it is optimized to store Volume Drives SCOs. It aggregates a set of OSDs like Alba's ASDs or Seagate's Kinetic Drives into a pool of storage that you can use as a storage backend for Open vStorage.
+
+Alba supports the following high-end set of features:
+* erasure coding
+* x-way replication
+* (optional) compression
+* (optional) encryption
+* partial reads
+* namespaces
+* [self healing](selfhealing.md) (repair, rebalancing)
+* ...
```

Comment:
This file also duplicates a lot of the current readme. You may want to remove some of the info from the current readme and just link to this one?